reactive/load_balancer.py

```diff
@@ -184,6 +185,17 @@ def upgrade_charm():
     maybe_write_apilb_logrotate_config()
 
 
+@hook('pre-series-upgrade')
+def pre_series_upgrade():
+    host.service_pause('nginx')
+    hookenv.status_set('blocked', 'Series upgrade in progress')
```

Comment:
Seems like this status could easily get overwritten by other handlers. I recommend setting it in an atexit handler or refactoring the charm to use layer-status.